apps/storybook/src/forms/Field.stories.tsx

```diff
@@ -12,6 +12,7 @@ type Story = StoryObj<typeof Field>;
 export const Basic: Story = {
   args: {
     children: <Input placeholder="This is an Input" />,
+    info: "Give a proper input",
```

Comment:
let's add a separate story just for `info` - and only have `label` in the Basic story

Reply:
let's remove info from the basic story
```suggestion
```